.github/workflows/release.yml

```diff
@@ -0,0 +1,65 @@
+name: Create release
+on: [push]
```

Comment:
```suggestion
on:
  push:
    branches:
      - 'master'
    paths:
      - 'pyproject.toml'
```
maybe?

Reply:
I dislike having the branch hardcoded, as I tend to use `main` for my branches, but I guess if people use the template they will get this name...

Reply:
```yml
if: format('refs/heads/{0}', github.event.repository.default_branch) == github.ref
```
maybe use this instead?

Reply:
That'll make it show up as a run but just be skipped. Your previous suggestion is the way to go I think

Reply:
can we use something like this?
```yaml
    branches:
      - ${{ github.event.repository.default_branch }}
```